src/components/ContentSidebar/ActivityFeed/activity-feed/EmptyState.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @flow
+ * @file Component for Activity feed empty state
+ */
+
+import React, { ReactNode } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import LoadingIndicator from 'box-react-ui/lib/components/loading-indicator/LoadingIndicator';
+import { IconActivityFeedEmptyState } from '../icons';
+
+import messages from '../messages';
+
+type Props = {
+    isLoading?: boolean,
+    showCommentMessage?: boolean
+};
+
+const EmptyState = ({ isLoading, showCommentMessage }: Props): ReactNode => (
+    <div className='box-ui-activity-feed-empty-state'>
+        <IconActivityFeedEmptyState />
+        {isLoading ? (
+            <LoadingIndicator />
+        ) : (
+            <div className='empty-state-cta'>
+                <b>
+                    <FormattedMessage {...messages.noActivity} />
```

Comment:
`<FormattedMessage tagName='b' {...messages.noActivity} />` instead of wrapping `<b>`